index.html

```diff
@@ -5010,11 +5010,16 @@ <h3>Co-signed</h3>
               <td>Individual</td>
               <td>Development; Open-Source Community Efforts, Documentation, Evangelism</td>
             </tr>	
-	     <tr>
+	          <tr>
```

Comment:
This small change seems unnecessary and should be removed.

Reply:
@pratikjain227 Sure will remove this.